src/Helpers/Utility/Timing.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Pantheon\Terminus\Helpers\Utility;
+
+class Timing
+{
+    private static $startTime;
+
+    /**
+     * Generate start time and assign to static class variable
+     */
+    public static function generateStartTime()
+    {
+        self::$startTime = new \DateTime();
+    }
+
+    /**
+     * Get the start time
+     *
+     * @return DateTime
+     */
+    public static function getStartTime()
+    {
+        if (empty(self::$startTime)) { // If startTime is not set, generate it
+            self::$startTime = self::generateStartTime();
```

Comment:
This is inconsistent with the implementation of generateStartTime(), which does not return a value. I might expect this to overwrite $startTime with null.

Reply:
I guess this line is never executed at the moment, since generateStartTime is called during preflight.

Reply:
Good catch. Thanks!